src/components/layout/Booking/BookingForm/BookingFormContent/BookingFormContent.jsx

```diff
@@ -137,11 +138,11 @@ class BookingFormContent extends PureComponent {
           </p>
         )}
 
-        {isEvent && (
+        {bookableTimes && hasBookableTimes && isEvent && (
           <p className="bc-retractation-delay">
```

Comment:
```suggestion
          <p className="bc-cancellation-delay">
```
Sinon on va avoir trop de dénomination pour la même notion métier derrière